elasticsearch/pom.xml

```diff
@@ -183,6 +183,7 @@
                 <jdk>9</jdk>
             </activation>
             <dependencies>
+                <!-- HSEARCH-3237: this should be removed when we remove the dependency from the Elasticsearch module to JAXB -->
```

Comment:
Why do we use JAXB in the Elasticsearch module? IIRC at some point we used it to format a date? But maybe we should get rid of it in 5.11 and be done with it?

Reply:
> But maybe we should get rid of it in 5.11 and be done with it?

Yep, exactly what I do in the following commits.